docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.2"
+services:
+  geth:
+    image: augurproject/dev-pop-geth:latest
+    ports:
```

Comment:
we don't want augurproject/dev-pop-geth:latest

we want the docker version that matches the augur-core version that augur.js uses. 

I think we need to call >npm explore augur.js -- npm run core:version to get the core-#### version of pop docker.

Reply:
@bthaile Getting this error:
```
✗ npm explore augur.js -- npm run core:version
npm ERR! missing script: core:version

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/justin/.npm/_logs/2018-04-09T12_49_14_561Z-debug.log
npm ERR! weird error 1
```

Reply:
```
npm explore augur.js -- npm run  --silent core:version
```
The added --silent suppresses the output.